### PR TITLE
Plasma refinery alt+click Fix

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request
Fixes a warning

Exact verbiage is now: "A sheet of solid plasma has been created."

## Why It's Good For The Game
Fix

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: fixed a on startup warning
/:cl:

### CODE REVIEW

**Review by itsmeow — CHANGES_REQUESTED** [review]
Remove unused variable P

File: code/modules/shuttle/super_cruise/shuttle_components/plasma_refiner.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	if(plasmoles >= 100)
 		var/obj/item/stack/sheet/mineral/plasma/P = new(src.loc, 1)
```
Comment (itsmeow) [inline code/modules/shuttle/super_cruise/shuttle_components/plasma_refiner.dm:42]: new /obj/item/stack/sheet/mineral/plasma(src.loc, 1)

**Comment by DatBoiTim** [comment]
> Remove unused variable P

No longer unused.

**Comment by itsmeow** [comment]
Amazing

**Review by itsmeow — APPROVED** [review]

**Comment by DatBoiTim** [comment]
Also better way to do that god forbid someone changes plasma to something else.

**Comment by itsmeow** [comment]
Speedmerging because I want to

**Pull request merged**